src/ports/postgres/modules/convex/mlp.sql_in

```diff
@@ -0,0 +1,736 @@
+loss           | 144.965776158
+num_iterations | 10000
+</pre>
+-# Now let's look at the prediction functions. In the following examples we will
+use the training data set for prediction as well, which is not usual but serves to
+show the syntax. First we will test the classification exampel.
+The prediction is in the the estimated_class_text column with the
+actual value in the class_text column.
+<pre class="example">
+SELECT mlp_predict(
+         'mlp_model',         -- Model table
+         'iris_data',         -- Test data table
+         'id',                -- Id column in test table
+         'mlp_prediction',    -- Output table for predictions
+         'response'           -- Output classes, not probabilities
+     );
+SELECT * FROM mlp_prediction JOIN iris_data USING (id);
+</pre>
+Result:
+<pre class="result">
+ id | estimated_class_text |    attributes     |   class_text    | class
+----+----------------------+-------------------+-----------------+-------
+  1 | Iris-setosa          | {5.1,3.5,1.4,0.2} | Iris-setosa     |     1
+  2 | Iris-setosa          | {4.9,3.0,1.4,0.2} | Iris-setosa     |     1
+  3 | Iris-setosa          | {4.7,3.2,1.3,0.2} | Iris-setosa     |     1
+  4 | Iris-setosa          | {4.6,3.1,1.5,0.2} | Iris-setosa     |     1
+  5 | Iris-setosa          | {5.0,3.6,1.4,0.2} | Iris-setosa     |     1
+  6 | Iris-setosa          | {5.4,3.9,1.7,0.4} | Iris-setosa     |     1
+  7 | Iris-setosa          | {4.6,3.4,1.4,0.3} | Iris-setosa     |     1
+  8 | Iris-setosa          | {5.0,3.4,1.5,0.2} | Iris-setosa     |     1
+  9 | Iris-setosa          | {4.4,2.9,1.4,0.2} | Iris-setosa     |     1
+ 10 | Iris-setosa          | {4.9,3.1,1.5,0.1} | Iris-setosa     |     1
+ 11 | Iris-versicolor      | {7.0,3.2,4.7,1.4} | Iris-versicolor |     2
+ 12 | Iris-versicolor      | {6.4,3.2,4.5,1.5} | Iris-versicolor |     2
+ 13 | Iris-versicolor      | {6.9,3.1,4.9,1.5} | Iris-versicolor |     2
+ 14 | Iris-versicolor      | {5.5,2.3,4.0,1.3} | Iris-versicolor |     2
+ 15 | Iris-versicolor      | {6.5,2.8,4.6,1.5} | Iris-versicolor |     2
+ 16 | Iris-versicolor      | {5.7,2.8,4.5,1.3} | Iris-versicolor |     2
+ 17 | Iris-versicolor      | {6.3,3.3,4.7,1.6} | Iris-versicolor |     2
+ 18 | Iris-versicolor      | {4.9,2.4,3.3,1.0} | Iris-versicolor |     2
+ 19 | Iris-versicolor      | {6.6,2.9,4.6,1.3} | Iris-versicolor |     2
+ 20 | Iris-versicolor      | {5.2,2.7,3.9,1.4} | Iris-versicolor |     2
+</pre>
+Prediction using the regression model:
+<pre class="example">
+SELECT mlp_predict(
```

Comment:
For consistency sake, should the reference to `mlp_predict` be `madlib.mlp_predict`?